parsers/GT.py

```diff
@@ -0,0 +1,90 @@
+# The arrow library is used to handle datetimes
+import arrow
+# The request library is used to fetch content through HTTP
+import requests
+# The pandas library is used to manipulate real time data
+import pandas as pd
+
+tz_gt = 'America/Guatemala'
+
+MAP_GENERATION = {
+    'coal': 'Turbina de Vapor',
+    'gas': 'Turbina de Gas',
+    'hydro': u'Hidroeléctrica',
+    'oil': 'Motor Reciprocante',
+    'solar': 'Fotovoltaica',
+    'wind': u'Eólico',
+    'geothermal': u'Geotérmica',
+}
+
+
+def fetch_production(country_code='GT', session=None):
+    #output frame
+    data = {
+        'countryCode': country_code,
+        'datetime': {},
```

Comment:
No need to declare an empty dict here. Confusing because it will be replaced by a python datetime object.